aiida/orm/data/cif.py

```diff
@@ -602,18 +642,65 @@ def has_partial_occupancies(self):
 
         return partial_occupancies
 
+    @property
     def has_attached_hydrogens(self):
         """
-        Check if there are hydrogens without coordinates, specified
-        as attached to the atoms of the structure.
-        :return: True if there are attached hydrogens, False otherwise.
+        Check if there are hydrogens without coordinates, specified as attached
+        to the atoms of the structure.
+
+        :returns: True if there are attached hydrogens, False otherwise.
         """
         tag = '_atom_site_attached_hydrogens'
         for datablock in self.values.keys():
             if tag in self.values[datablock].keys():
                 for value in self.values[datablock][tag]:
                     if value != '.' and value != '?' and value != '0':
                         return True
+
+        return False
+
+    @property
+    def has_atomic_sites(self):
+        """
+        Returns whether there are any atomic sites defined in the cif data. That
+        is to say, it will check all the values for the `_atom_site_fract_*` tags
+        and if they are all equal to `?` that means there are no relevant atomic
+        sites defined and the function will return False. In all other cases the
+        function will return True
+
+        :returns: False when at least one atomic site fractional coordinate is not
+            equal to `?` and True otherwise
+        """
+        tag_x = '_atom_site_fract_x'
+        tag_y = '_atom_site_fract_y'
+        tag_z = '_atom_site_fract_z'
+        coords = []
+        for datablock in self.values.keys():
+            for tag in [tag_x, tag_y, tag_z]:
+                if tag in self.values[datablock].keys():
+                    coords.extend(self.values[datablock][tag])
+
+        return not all([coord == '?' for coord in coords])
+
+    @property
+    def has_unknown_species(self):
+        """
+        Returns whether the cif contains atomic species that are not recognized by AiiDA.
+        The known species are taken from the elements dictionary in aiida.common.constants.
+        If any of the formula of the cif data contain species that are not in that elements
+        dictionary, the function will return True and False in all other cases
+
+        :returns: True when there are unknown species in any of the formulae
+        """
+        from aiida.common.constants import elements
+
+        known_species = [element['symbol'] for index, element in elements.items()]
```

Comment:
known_species = [element['symbol'] for element in elements.values()]

I know, I'm picky today...